src/static/index.js

```diff
@@ -151,23 +151,20 @@ function watchDeviceEvents(params) {
 
 function leaveDeviceRoom() {
   astarteClient.joinedRooms().forEach((room) => {
-    console.log(`leaving room ${room}`);
     astarteClient.leaveRoom(room);
   });
 }
 
 function loadPage(page) {
   const elem = document.getElementById('react-page');
   if (elem) {
-    console.log('React already initialized, skipping');
```

Comment:
Logging to the console can actually be useful in this file. How about we keep using `console.log` but just in the Dev environment?

Reply:
This was just for debugging purposes. Once tested right, this should have been removed.